### PR TITLE
AND-3733 | implement mute unmute api

### PR DESCRIPTION
**Pull request by hchauhansf**
In this PR,
I have implemented mute/unmute and video speaker apis.
I have solved one of the crash getting for foreground service notification and solved the issue of notification not getting dismissed after the incoming or missed call.

### CODE REVIEW

File: sipservice/src/main/java/net/gotev/sipservice/SipService.java [hunk 1]
```diff
@@ -380,6 +381,10 @@ private void handleSendDTMF(Intent intent) {
             try {
                 SipUtility.playSound(dtmf + ".wav", this.getApplicationContext());
                 sipCall.dialDtmf(dtmf);
+                if (dtmf.equals("9")) {
```
Comment (pratik-pdc) [inline sipservice/src/main/java/net/gotev/sipservice/SipService.java]: Replace Hard-coded value either with enum or constants

File: sipservice/src/main/java/net/gotev/sipservice/SipService.java [hunk 2]
```diff
@@ -624,6 +647,7 @@ private void handleAttendedTransferCall(Intent intent) {
     }
 
     private void handleSetIncomingVideoFeed(Intent intent) {
+        startForeground(121, createForegroundServiceNotification(this, getString(R.string.app_name)));
```
Comment (pratik-pdc) [inline sipservice/src/main/java/net/gotev/sipservice/SipService.java]: Notification ID replace with constant

File: sipservice/src/main/java/net/gotev/sipservice/SipService.java [hunk 3]
```diff
@@ -649,6 +673,7 @@ private void handleSetIncomingVideoFeed(Intent intent) {
                 sipCall.setIncomingVideoFeed(surface);
             }
         }
+        enqueueDelayedJob(() -> stopForeground(false), 200);
```
Comment (pratik-pdc) [inline sipservice/src/main/java/net/gotev/sipservice/SipService.java]: Move delay value to constant

File: sipservice/src/main/java/net/gotev/sipservice/SipService.java (hunk 1 above)
Comment (hchauhansf) [reply sipservice/src/main/java/net/gotev/sipservice/SipService.java]: Done, added the changes in this PR

File: sipservice/src/main/java/net/gotev/sipservice/SipService.java (hunk 2 above)
Comment (hchauhansf) [reply sipservice/src/main/java/net/gotev/sipservice/SipService.java]: Done, added the changes in this PR

File: sipservice/src/main/java/net/gotev/sipservice/SipService.java (hunk 3 above)
Comment (hchauhansf) [reply sipservice/src/main/java/net/gotev/sipservice/SipService.java]: Done, added the changes in this PR

**Review by pratik-pdc — APPROVED** [review]
Good to go!

**Pull request merged**